dhcpv6/dhcpv6message.go

```diff
@@ -198,26 +198,28 @@ func NewRequestFromAdvertise(advertise DHCPv6, modifiers ...Modifier) (DHCPv6, e
 	return d, nil
 }
 
-// NewReplyFromRebind creates a new REPLY packet based on a REBIND packet.
-func NewReplyFromRebind(rebind DHCPv6, modifiers ...Modifier) (DHCPv6, error) {
-	if rebind == nil {
-		return nil, errors.New("REBIND cannot be nil")
+// NewReplyFromDHCPv6Message creates a new REPLY packet based on a DHCPv6Message.
```

Comment:
Please add some more information in the comment, in particular that this function is to be used when generating a reply to REQUEST, CONFIRM, RENEW, REBIND and RELEASE